.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
 <!--- Provide a brief summary of the issue in the title above -->
+<!--- We'd kindly ask you to open any issues related to the Gradle Android plugin or Android studio on the Google forum at `https://source.android.com/source/report-bugs` -->
```

Comment:
I just thought maybe it stresses the point that there is a specific section for **Android+Gradle** section in that Issue Tracker, and that is not only about general Android stuff. But it's probably, as you say, too much detail.

I am fine with not adding that.